nss-day-cohort-41/tabloid-fullstack-shocking-secret-keepers
# Th post/user posts

# Description
Added the post index, user post index, post details, and post add.
Fixes # (issue)
## Type of change
Please delete options that are not relevant.

# Testing Instructions
Git fetch --all
git checkout into th-Post/UserPosts
Login to an admin account
Go to the posts menu option and all of the posts should populate.
Go to User Posts menu option and it should show only posts added by the user.
Click on the title of any post to see the post details page.
There is an add button at the top of posts and user posts.
Click this button to go to the add post form.
Fill out the forms and submit and it should show in both post lists now.
# Checklist:
